### ISSUE TITLE
Improve color contrast ratio of certain text items in Web Modeler

### ISSUE THREAD

**Issue by lmbateman** [camunda/camunda-modeler#4625]
### Problem you would like to solve

There are a few places in the Web Modeler where the contrast doesn't meet WCAG guidelines. Technically the contrast guidelines are AA, so we're not obligated to meet them, but if it's easy to fix, then we should.

### Proposed solution

Increase the contrast of the indicated text in the attached screenshots. Ideally, the contrast ratio for 14px text should be 4.5:1.

### Alternatives considered

Leave the text inaccessible. 

### Additional context

![image](https://github.com/user-attachments/assets/6802ac65-7509-4471-b0ec-c801db74dd5f)
![image](https://github.com/user-attachments/assets/017012cb-06d1-4588-84c5-4cb4e3089959)
![image](https://github.com/user-attachments/assets/79b6e5b0-ca86-4efc-8ce0-63003369232a)
![image](https://github.com/user-attachments/assets/b10e157b-b7e3-41d0-bdf6-b68646a6e256)
